src/test/java/com/wikia/webdriver/common/core/drivers/browsers/GhostBrowser.java

```diff
@@ -0,0 +1,39 @@
+package com.wikia.webdriver.common.core.drivers.browsers;
+
+import java.io.File;
+
+import org.openqa.selenium.phantomjs.PhantomJSDriver;
+import org.openqa.selenium.phantomjs.PhantomJSDriverService;
+
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.drivers.BrowserAbstract;
+
+public class GhostBrowser extends BrowserAbstract {
+
+  @Override
+  public void setOptions() {
+    String phantomJSBinaryName;
+    String OsName = System.getProperty("os.name").toUpperCase();
+
+    if (OsName.contains("WINDOWS")) {
+      phantomJSBinaryName = "phantomjs.exe";
+
+      File phantomJSBinary =
+          new File("." + File.separator + "src" + File.separator + "test" + File.separator
+              + "resources" + File.separator + "PhantomJS" + File.separator + phantomJSBinaryName);
+
+      caps.setCapability(PhantomJSDriverService.PHANTOMJS_EXECUTABLE_PATH_PROPERTY,
+          phantomJSBinary.getAbsolutePath());
+    }
+  }
+
+  @Override
+  public WikiaWebDriver create() {
+    return new WikiaWebDriver(new PhantomJSDriver(caps), false);
+  }
+
+  @Override
+  public void addExtension(String extensionName) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1186)